expiringdict/__init__.py

```diff
@@ -110,22 +114,36 @@ def get(self, key, default=None, with_age=False):
     def items(self):
         """ Return a copy of the dictionary's list of (key, value) pairs. """
         r = []
-        for key in self:
-            try:
-                r.append((key, self[key]))
-            except KeyError:
-                pass
+        if sys.version_info >= (3, 5):
```

Comment:
Yes, that's certainly a better solution, but the lambda function will give TypeError under python 3. I will try to solve this.